StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/AccountPickerRowView.swift

```diff
@@ -66,6 +66,7 @@ final class AccountPickerRowView: UIView {
 
     init(
         isDisabled: Bool,
+        isFaded: Bool,
```

Comment:
this was introduced because for this new use case we want to:
1) fade (or "gray out") the row 
2) ...but we DON'T want to disable taps (...because we want to display the drawer)